sdks/java/extensions/zetasketch/build.gradle

```diff
@@ -0,0 +1,64 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * License); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import groovy.json.JsonOutput
+
+plugins { id 'org.apache.beam.module' }
+applyJavaNature()
+
+description = "Apache Beam :: SDKs :: Java :: Extensions :: ZetaSketch"
+
+def zetasketch_version = "0.1.0"
+
+dependencies {
+    compile library.java.vendored_guava_26_0_jre
+    compile project(path: ":sdks:java:core", configuration: "shadow")
+    compile "com.google.zetasketch:zetasketch:$zetasketch_version"
+    testCompile library.java.junit
+    testCompile project(":sdks:java:io:google-cloud-platform")
+    testRuntimeOnly project(":runners:direct-java")
+    testRuntimeOnly project(":runners:google-cloud-dataflow-java")
+}
+
+/**
+ * Integration tests running on Dataflow with BigQuery.
+ */
+task integrationTest(type: Test) {
+    group = "Verification"
+    def gcpProject = project.findProperty('gcpProject') ?: 'apache-beam-testing'
+    def gcpTempRoot = project.findProperty('gcpTempRoot') ?: 'gs://temp-storage-for-end-to-end-tests'
+    systemProperty "beamTestPipelineOptions", JsonOutput.toJson([
+            "--runner=TestDataflowRunner",
+            "--project=${gcpProject}",
+            "--tempRoot=${gcpTempRoot}",
+    ])
```

Comment:
Within current configuration, the test will run with the default worker image, not the HEAD worker code. Would you like to explain more about why choose worker image rather than HEAD?

Reply:
I am using the same setup as integration tests for the `sdks:java:io:google-cloud-platform` module, which includes other integration tests with BigQuery (https://github.com/apache/beam/blob/c0f456f7aab8d329ae13b4c6b606f6ad85de9619/sdks/java/io/google-cloud-platform/build.gradle#L85).

What specific changes to the setup config are you suggesting?

Reply:
But you want to run our test with DataflowRunner right? Then if you want to always run with the worker head, you need to add 2 more cmd args like:
https://github.com/apache/beam/blob/master/runners/google-cloud-dataflow-java/build.gradle#L161. Otherwise, the prebuilt worker image is used.

Reply:
Thanks for the pointer. Will do.

Reply:
Done.